### DISCUSSION TITLE
Can't fix broken buckets because of "cause(loadBucketMetadata: unknown format: 56541)"

### DISCUSSION THREAD

**Discussion by pozylon** [minio/minio#14458] [Q&A] [answered]
Due to a failed update and a node going offline, 2 of my buckets went into "limbo":
<img width="641" alt="Bildschirmfoto 2022-03-03 um 08 59 02" src="https://user-images.githubusercontent.com/2109393/156521627-12c1350b-1d0b-475e-b639-73134dcac0b2.png">

I've seen the following effects on those buckets: 
- DELETE, PUT or POST operations don't work but GET does
- It shows a creation time of 0001-01... (see screenshot)
- It's not possible to remove the bucket through mc admin as it's not possible to get the list of ARN's for that bucket
- It's not possible to remove the bucket through mc rb even when trying dangerous and force
- When trying to get the events of the Bucket through the Web Console, an Error pops up: We encountered an internal error, please try again.: **cause(loadBucketMetadata: unknown format: 56541)** I think that is the root cause of all the operations failing

How can I finally get rid of those broken buckets and free up the space?

I'm running the latest version: 2022-02-26

**Comment by klauspost** [comment, marked as answer]
You can try  shut down minio, remove (or initially move) `.minio.sys/<bucket>` and `<bucket>` on all drives. Restart.

**Comment by harshavardhana** [comment]
> Due to a failed update and a node going offline, 2 of my buckets went into "limbo":

care to explain more ?
